### DISCUSSION TITLE
How to install in a production env [May 26]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2247] [Discord Threads]
> 🙋‍♂️ **jiub**

**Comment by o1lab** [comment]
> 🙋‍♂️ **FrancyMak**  
#### Hi!  
#  
> 🙋‍♂️ **FrancyMak**  
#### You have different ways  
#  
> 🙋‍♂️ **FrancyMak**  
#### You can easily install noco through Node.  
#  
> 🙋‍♂️ **FrancyMak**  
#### https://docs.nocodb.com/getting-started/installation#node-application  
#  
> 🙋‍♂️ **FrancyMak**  
#### that's one of the simplest way  
#  
> 🙋‍♂️ **FrancyMak**  
#### if you don't have Node or you have a not updated version in your machine. you can download it with these commands:   
####    
#### ```bash   
#### sudo curl -sL https://deb.nodesource.com/setup_14.x | sudo -E bash -   
####    
#### sudo apt install nodejs   
####    
#### sudo apt install build-essential   
####    
#### node --version   
#### ```  
#  
> 🙋‍♂️ **wkw**  
#### Alternatively, most ppl choose docker. See https://docs.nocodb.com/getting-started/installation#docker  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/978983274170945566)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
